src/i18n/locales/en/translation.json

```diff
@@ -476,7 +476,7 @@
         "heading": "Do you want to login to"
       },
       "submit": "Login",
-      "success": "Authenticated successfully",
+      "success": "Authenticated successfully into {{name}}",
```

Comment:
Is "with" better than "into"? @reneaaron

Reply:
"Authenticated into" feels wrong to me. How about "Login successful on {{name}}"?

Reply:
Done!